examples/pipe/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
```

Comment:
Added this to make it easier for newcomers to start a local Kafka and run the samples.